chapters/00-why-should-i-learn.md

```diff
@@ -50,4 +56,15 @@ You can take a look at the [awesome nvim plugin list](https://github.com/rockerB
 
 ---
 
+# How should I start use nvim?
+nvim has a steep learning curve it takes time and patience to learn, you will find that you need to invest time to **start** using nvim as your daily driver.
+
+If you are a professional programmer I recommend to start use it in your personal projects and integrate it slowly to your professional work, I did it by having my previous editor (Sublime Text) open while nvim was open too.
+
+It started by trying to do simple stuff with nvim and moving to sublime when I need to do complicated stuff I struggled with nvim, after I got comfortable with the simple stuff I tried to do complicated stuff with nvim, sometimes I succeed sometimes I failed and went to sublime, after a while I felt more comfortable with nvim and sticked with it all day long.
+
+The bottom line is it will take time and effort but eventually it will be worth it, for me it sparked the old feeling of learn how to edit your code, and for now it still on everyday.
```

Comment:
```suggestion
The bottom line is that it'll take time and effort but eventually it will be worth it, for me it sparked the old feeling of learning how to edit your code, and I still feel it every day.
```